whois-nrtm4/src/main/java/net/ripe/db/nrtm4/util/NrtmFileUtil.java

```diff
@@ -17,6 +17,11 @@ public static String newFileName(final PublishableNrtmDocument file) {
         return String.format("%s.%d.%s.json", prefix, file.getVersion(), randomHexString());
```

Comment:
newFileName visibility could be private, right? we are just using it from newGzFileName